scripts/code_style.py

```diff
@@ -56,8 +56,7 @@ def get_uncrustify_version() -> str:
     """
     Get the version string from Uncrustify
     """
-    version_args = ["--version"]
-    result = subprocess.run([UNCRUSTIFY_EXE] + version_args, \
+    result = subprocess.run([UNCRUSTIFY_EXE, "--version"], \
```

Comment:
I should have made it clearer initially, but my comment about removing explicit line continuation was about all cases where it's not needed. (I'm actually not sure a single one is needed in this script.)